src/qibo/models/variational.py

```diff
@@ -473,6 +572,7 @@ def minimize(
             method (str): the desired minimization method.
                 See :meth:`qibo.optimizers.optimize` for available optimization
                 methods.
+            mode (str): the desired loss function.
```

Comment:
Please include the list of possible options so the user does not have to read the code, e.g. `cvar`, `gibbs`.